gprofiler/main.py

```diff
@@ -769,7 +785,6 @@ def parse_cmd_args() -> configargparse.Namespace:
     )
 
     args = parser.parse_args()
-
```

Comment:
Bring back :)

Reply:
It was never deleted, not sure why git shows this deletion here :o

Reply:
In master it looks like:
```
    args = parser.parse_args()

    args.perf_inject = args.nodejs_mode == "perf"
    args.perf_node_attach = args.nodejs_mode == "attach-maps"
```

how does it look in your branch?

Reply:
Oh, ok. I changed it back.